client/src/components/AdvisorReports/AdvisorReportsRow.js

```diff
@@ -1,21 +1,22 @@
 import Checkbox from "components/Checkbox"
-import _uniqueId from "lodash/uniqueId"
 import PropTypes from "prop-types"
 import React from "react"
 
 const _advisorStats = (columnGroups, statistics) => {
   const stats = []
-  columnGroups.forEach(group => {
+  columnGroups.forEach((group, index) => {
     let rowCell = statistics.filter(s => s.week === group)
```

Comment:
`groups` are already unique values, why do we use index which can have side-effects later down the road? I suggest to use `key={`submitted_${group}`}` and remove `index`